src/current/_includes/releases/v23.1/v23.1.11.md

```diff
@@ -0,0 +1,137 @@
+- Added a new gauge [metric](../v23.1/metrics.html) `sql.schema.invalid_objects`. This gauge is periodically updated based on the schedule set by the `sql.schema.telemetry.recurrence` [cluster setting](../v23.1/cluster-settings.html). When the metric is updated, it counts the number of schema objects ([tables](../v23.1/create-table.html), [types](../v23.1/create-type.html), [schemas](../v23.1/create-schema.html), [databases](../v23.1/create-database.html), and [functions](../v23.1/user-defined-functions.html)) that are in an invalid state according to CockroachDB's internal validation checks. This metric is expected to be zero (`0`) in a healthy cluster. If it is not zero, it indicates that there is a problem that must be repaired. [#109733][#109733]
+- Added two new [changefeed metrics](../v23.1/monitor-and-debug-changefeeds.html#recommended-changefeed-metrics-to-track): `changefeed.checkpoint_progress`  is similar to `changefeed.max_behind_nanos` but supports [metrics labels](../v23.1/monitor-and-debug-changefeeds.html#using-changefeed-metrics-labels); `changefeed.aggregator_progress` tracks the progress of individual aggregators (the lowest timestamp for which all aggregators with the label have emitted all values they're responsible for). [#109744][#109744]
+
+<h3 id="v23-1-11-command-line-changes">Command-line changes</h3>
+
+- Removed the command `\demo recommission` from [`cockroach demo`]({% link v23.1/cockroach-demo.md %}). It had been obsolete and non-functional ever since v20.2. [#108631][#108631]
+
+<h3 id="v23-1-11-db-console-changes">DB Console changes</h3>
+
+- Users without the `VIEWCLUSTERSETTINGS` [permission]({% link v23.1/security-reference/authorization.md %}#supported-privileges) but with the `VIEWACTIVITY` or `VIEWACTIVITYREDACTED` permissions can now see [index recommendations]({% link v23.1/ui-databases-page.md %}#index-recommendations). [#109466][#109466]
+- CockroachDB will now show a warning when the time period selected on the [SQL Activity page](../v23.1/ui-overview.html#sql-activity) is older than the oldest data available. [#109468][#109468]
+
+<h3 id="v23-1-11-bug-fixes">Bug fixes</h3>
+
+- Fixed a buggy TTL descriptor repair by removing it. Previously, upgrading from v22.2.* to 23.1.9 incorrectly removed [TTL storage params]({% link v23.1/row-level-ttl.md %}##ttl-storage-parameters) from tables (visible via [`SHOW CREATE TABLE  <ttl-table>`]({% link v23.1/show-create-table.md %})) while attempting to repair table descriptors. This resulted in the node that attempts to run the TTL [job](../v23.1/show-jobs.html) crashing due to a panic caused by the missing TTL storage parameters. Clusters currently on v22.2.* should **NOT** be upgraded to v23.1.9 and should be upgraded directly to v23.1.10 or later. [#110562][#110562]
```

Comment:
Should the failing link be https://www.cockroachlabs.com/docs/stable/show-create#show-the-create-table-statement-for-a-table ?

Reply:
ah thanks! should be fixed in latest versions